core/src/main/java/hudson/model/UpdateSite.java

```diff
@@ -528,21 +528,6 @@ public String getDownloadUrl() {
         return url;
     }
 
-    /**
-     * Is this the legacy default update center site?
-     */
-    public boolean isLegacyDefault() {
```

Comment:
This method is overridden in https://github.com/jenkinsci/simpleupdatesite-plugin , and hence it will lead to code compatibility issues. Not sure anyone cares

Reply:
I'm OK with dropping support for this plugin as long as a deprecation notice is added to the Update Center and an entry is added to the changelog/upgrade guide.

Reply:
> hence it will lead to code compatibility issues

Why does this break?

Reply:
The `@Override` definition in simpleupdatesite-plugin will prevent the code from compiling against new Jenkins core versions. There should be no other consequences